cleanlab_studio/internal/enrichment_utils.py

```diff
@@ -91,8 +91,8 @@ def get_optimized_prompt(prompt: str, constrain_outputs: Optional[List[str]] = N
     """
 
     if constrain_outputs is not None:
-        string_constrain_outputs = str(constrain_outputs).replace("'", "")
-        pre_prompt = f"Your answer must exactly match one of the following values: [{string_constrain_outputs}].\n"
+        string_constrain_outputs = ", ".join(constrain_outputs)
+        pre_prompt = f"Your answer must only contain one of the following values: [{string_constrain_outputs}].\n"
```

Comment:
Do we still want to leave in the list wrapped around the string_constrain_outputs here?

Reply:
The current prompt template is:
```
Your answer must only contain one of the following values: [choice1, choice2].
```

Are you suggesting that we should remove the square braces all together? (this change so far has updated it from double square braces -> single)

Reply:
I think single list should be fine but also didn't test it without any brackets, I can always try without them locally though